torch/_dynamo/variables/torch.py

```diff
@@ -1133,5 +1133,6 @@ def _nn_param_via_prefix_insert(tx: "InstructionTranslator", data, requires_grad
     def torch_function_override_enabled(self, tx, args, kwargs):
         return (
             self.get_function() in get_overridable_functions()
+            or self.get_function().__module__.startswith("torch._ops")
```

Comment:
Can you check if the function isinstance(self.get_function(), (torch.ops.OpOverload, torch.OpOverloadPacket)) instead? There are things in torch.ops that aren't actually overridable (like torch.ops.load_library)